thundra/integrations/modules/sqlite3.py

```diff
@@ -0,0 +1,65 @@
+import wrapt
+
+from thundra.integrations.sqlite3 import SQLiteIntegration
+from thundra.config import config_names
+from thundra.config.config_provider import ConfigProvider
+
+sqlite_integration = SQLiteIntegration()
+
+
+class SqliteCursorWrapper(wrapt.ObjectProxy):
+
+    def __init__(self, cursor, connection_wrapper):
+        super(SqliteCursorWrapper, self).__init__(cursor)
+        self._self_connection = connection_wrapper
+
+    def execute(self, *args, **kwargs):
+        return sqlite_integration.run_and_trace(
+            self.__wrapped__.execute,
+            self._self_connection,
+            args,
+            kwargs,
+        )
+
+    def __enter__(self):
+        # raise appropriate error if api not supported (should reach the user)
+        self.__wrapped__.__enter__
+        return self
+
+
+class SqliteConnectionWrapper(wrapt.ObjectProxy):
+    db_name = None
+    host = None
+
+    def __init__(self, connection, db_name, host='localhost'):
```

Comment:
who pass a value for `host` arg?

Reply:
Is it possible to get it from `wrapper` args as you did for `db_name`?

Reply:
there is no "host" information in "args" or "kwargs" parameters.